test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -421,36 +465,17 @@ UCS_TEST_P(test_ucp_peer_failure, bcopy_multi, "SEG_SIZE?=512", "RC_TM_ENABLE?=n
             false /* must_fail */);
 }
 
-UCS_TEST_P(test_ucp_peer_failure, force_close, "RC_FC_ENABLE?=n") {
+UCS_TEST_P(test_ucp_peer_failure, force_close, "RC_FC_ENABLE?=n",
+           /* To catch unexpected descriptors leak, for multi-fragment protocol
+              with TCP */
+           "TCP_RX_SEG_SIZE?=1024", "TCP_TX_SEG_SIZE?=1024")
+{
     do_test(16000, /* msg_size */
             1000, /* pre_msg_cnt */
             true, /* force_close */
             false /* must_fail */);
 }
 
-UCS_TEST_SKIP_COND_P(test_ucp_peer_failure, disable_sync_send,
```

Comment:
this test is no longer relevant is we don't disable sync protocol - we have a method to track outstanding requests